testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GradleBuildIsolatedProjectsSmokeTest.groovy

```diff
@@ -91,8 +91,7 @@ class GradleBuildIsolatedProjectsSmokeTest extends AbstractGradleBuildIsolatedPr
                 "Project ':' cannot access 'Project.plugins' functionality on subprojects via 'allprojects'",
                 "Project ':' cannot access 'Project.extensions' functionality on subprojects via 'allprojects'",
             )
-            // maximum number of problems we collect (should be 86520)
```

Comment:
This was what made us notice the issue with lack of deduplication. I think 86520 was what I saw with a previous version of the init script used by this test which would fail to schedule some tasks.